### PR TITLE
Gatsby 2.27

### PR DESCRIPTION
**Pull request by alexfornuto**
## Summary

**Site Upgrade** - Upgrade to Gatsby 2.27. Note: the development server should be started with `npm run develop` to include the flags for experimental new features.

## Post Launch

- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by alexfornuto** [comment]
This needs an updated docker image from the orb providing Node >= 11, waiting on that.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-6146](https://pr-6146--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `60` | `61` | `1` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-6146` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/25182/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/25182/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-6146--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by alexfornuto** [comment]
Blocked by https://github.com/pantheon-systems/frontend-orb/pull/53, as we want to keep using versions from the default branch for the orb.

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 orbs:
-  pfe: pantheon-systems/front-end@0.3.0
+  pfe: pantheon-systems/front-end@dev:node-12
```
Comment (georch) [inline .circleci/config.yml:4]:
```suggestion
  pfe: pantheon-systems/front-end@0.4.0
```

**Review by georch — CHANGES_REQUESTED** [review]
@alexfornuto The team published a new release to update to Node 12 https://github.com/pantheon-systems/frontend-orb/releases/tag/0.4.0 Please update the CircleCI accordingly. cc @carolyn-shannon

File: .circleci/config.yml (hunk 1 above)
Comment (carolyn-shannon) [reply .circleci/config.yml]: Thanks, will do.

**Comment by jmolivas** [comment]
This PR can be closed since the PR https://github.com/pantheon-systems/documentation/pull/6505 was merged cc: @carolyn-shannon @EdwardAngert

**Comment by carolyn-shannon** [comment]
Duplicate of #6505

**Pull request closed without merging**